SetupTearDown.ps1

```diff
@@ -118,16 +118,9 @@ param(
         Write-Host "`nSSHD default config files already exist"
     }
 
-    if (-NOT (Test-Path "$env:ProgramData\ssh\administrators_authorized_keys"))
-    {
-        Write-Host "`nAdd the AuthorizedKey as a trusted admin key"
-        Add-Content -Force -Path "$env:ProgramData\ssh\administrators_authorized_keys" -Value "$authorizedKey"
-        icacls.exe "$env:ProgramData\ssh\administrators_authorized_keys" /inheritance:r /grant "Administrators:F" /grant "SYSTEM:F"
-    }
-    else
-    {
-        Write-Host "`nTrusted admin keys already exist"
-    }
+    Write-Host "`nAdd the AuthorizedKey as a trusted admin key"
+    Add-Content -Force -Path "$env:ProgramData\ssh\administrators_authorized_keys" -Value "`n$authorizedKey"

```

Comment:
Do you know if it will cause problems if we add the same key multiple times? I think after this change we may add the same key multiple times to the authorized keys file. Maybe it would be better to see if the key is present in the file before adding it.

Also, are you hitting this during manual or automated end-to-end test passes? Since we're recreating the containers for every test, you shouldn't hit this issue for the full test passes.

Reply:
It’s for the external setup. I can check if the passkey already exists for better code cleanup.